src/pybind/mgr/dashboard/controllers/ceph_users.py

```diff
@@ -69,6 +69,20 @@ def user_create(_, user_entity: str = '', capabilities: Optional[List[Cap]] = No
 
         return f"Successfully created user '{user_entity}'"
 
+    @staticmethod
+    def get_or_create(_, user_entity: str = '', capabilities: Optional[List[Cap]] = None):
```

Comment:
I think we can consider this [command](https://docs.ceph.com/en/latest/rados/operations/user-management/#getting-a-user) instead.

Reply:
@afreen23 get is not returning the credentials. My goal is to have an endpoint to get the user credentials.

Reply:
@afreen23, what can I do to help on this?